bitcoin/src/blockdata/script/tests.rs

```diff
@@ -213,12 +212,12 @@ fn script_generators() {
     assert!(ScriptBuf::new_v0_p2wpkh(&wpubkey_hash).is_v0_p2wpkh());
 
     let script = Builder::new().push_opcode(OP_NUMEQUAL).push_verify().into_script();
-    let script_hash = ScriptHash::hash(&script.serialize());
+    let script_hash = ScriptHash::hash(&script.to_bytes());
     let p2sh = ScriptBuf::new_p2sh(&script_hash);
     assert!(p2sh.is_p2sh());
     assert_eq!(script.to_p2sh(), p2sh);
 
-    let wscript_hash = WScriptHash::hash(&script.serialize());
+    let wscript_hash = WScriptHash::hash(&script.to_bytes());
```

Comment:
Concept ACK removing dependency on `psbt` but simply calling `script.script_hash()` would be better.

Reply:
What! Actually pay attention to the line of code I'm changing when doing so - outrageous (read: sarcasm about my own sloppy work :)